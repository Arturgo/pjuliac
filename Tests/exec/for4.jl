
# la variable de la boucle for est locale
# mais les bornes sont évaluées dans l'environnement global

function f(x)
    print(x, ".")
    x
end

n = 42
println(n)

for n = f(1) : f(n)
    print(n)
end
println()

println(n)

