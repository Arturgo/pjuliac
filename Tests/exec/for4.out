42
1.42.123456789101112131415161718192021222324252627282930313233343536373839404142
42
